### PR TITLE
WWW-218: Microjourney Feedback button, WWW-191: Calc Input Update

### PR DESCRIPTION
**Pull request by colbytcook**
## Jira

- https://pegadigitalit.atlassian.net/browse/WWW-218
- https://pegadigitalit.atlassian.net/browse/WWW-191

## Summary

- Removed feedback button
- Added input pattern to the calculator form

## Details

- Removed the feedback button from the micorjourney Admin expe…rience
- Added a pattern to the calculator input fields to remove the addition of a decimal

## How to test
- Pull down this branch
- Navigate to "/pattern-lab/?p=experiments-step-2-demo"
- Confirm there is no "Feedback" button showing on the bottom right of the screen

### CODE REVIEW

File: docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig [hunk 1]
```diff
@@ -69,6 +69,7 @@
                     required: true,
                     type: "number",
                     id: for_value,
+                    pattern: '/\d/',
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:72]: Double quotes in twig, please. :)

Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:72]: sure thing!

Comment (colbytcook) [reply docs-site/src/pages/pattern-lab/_patterns/50-pages/55-d8-calculator/00-d8-calculator.twig:72]: @mikemai2awesome updated

**Review by remydenton — APPROVED** [review]
As a general rule, create separate PRs, or at least separate commits, for unrelated functionality.

That said, this is pretty simple and you did cleanup, so approved 😄

**Review by mikemai2awesome — APPROVED** [review]

**Review by danielamorse — APPROVED** [review]
@colbytcook confirmed there's no feedback button. Is there anything calculator-related to test or is that not necessary?

**Comment by colbytcook** [comment]
@danielamorse not really, that pattern code was already sent out to the Drupal team to update on there end, the addition here was just to attempt to keep things inline

**Pull request merged**